### PR TITLE
feat: Add exit/error signal

### PR DESCRIPTION
**Pull request by enicio**
**What was done to achieve this purpose?**
    handles the azion-framework-manager exit code and then stops the process on error
**How to test if this really works?**
   Manual test with axioncli publish and expect process stop on error
**Who can help review this Merge Request?**
    @vinigfer
**Is closing some issue?**
    Yes

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+_Doing the deploy with azioncli and receive the expected errors_
+
+**Who can help review this Merge Request?**
+
+_Who are the best people to review this Merge Request?_
+
+**Is closing some issue?**
+
+_No._
```
Comment (vinigfer) [inline .github/pull_request_template.md:19]: Actually we should probably keep this with comments, and fill when opening a PR right?
```suggestion
<!--- Fix issue xyz -->

**What was done to achieve this purpose?**
<!-- Describe the key points of implementation  -->

**How to test if this really works?**
<!-- Manual and automated tests that can be executed  -->

**Who can help review this Merge Request?**
<!-- Suggest with @ people inside the Contributors section  -->

**Is closing some issue?**
<!-- Yes/No  -->
```

Comment (enicio) [reply .github/pull_request_template.md:19]: sorry, i misunderstood the usage of PR template. :)

File: webdev/nextjs/webdev.sh [hunk 2]
```diff
@@ -131,6 +131,6 @@ case "$1" in
         cd azion/cells-site-template || exit $?
         # Publish only assets
         azion-framework-adapter publish --config ../kv.json \
-                               --only-assets --assets-dir ../../out
+                               --only-assets --assets-dir ../../out || exit $?
         echo "{}" > ./args.json;;
```
Comment (MagnunAVFAzion) [inline webdev/nextjs/webdev.sh]: Probably we should move this command `echo "{}" > ./args.json` to build step (after line 121).

**Review by MagnunAVFAzion — APPROVED** [review]
lgtm 🤘

**Pull request merged**